src/_includes/layouts/handbook.njk

```diff
@@ -71,55 +71,5 @@ date: git Last Modified
     </div>
 </div>
 </div>
-<script>
```

Comment:
I've moved it to the `common-js.njk` partial, since both that script and the partial are only used in the `handbook` and `learning-resources` template